src/Files.App/Actions/FileSystem/RenameAction.cs

```diff
@@ -33,15 +33,22 @@ public RenameAction()
 		}
 
 		public Task ExecuteAsync(object? parameter = null)
-		{
-			context.ShellPage?.SlimContentPage?.ItemManipulationModel.StartRenameItem();
+		{	
+			if (context.SelectedItems.Count == 1)
```

Comment:
Is this if statement needed? It looks like the else statement has the same logic.